src/assets/scss/components/ModelSelection.scss

```diff
@@ -1,17 +1,17 @@
 
-// .dropdown {
-//     .pf-c-dropdown {
-//         &__menu {
-//             &--before {
-//                 border-color: #06c;
-//             }
-//         }
-//     }
+.dropdown {
```

Comment:
Adding this for visibility sake, would prefer not to keep this though because it won't be standardized.